### PR TITLE
[TravisCI] Add eoan build.

### PR DESCRIPTION
**Pull request by fghoussen**
## Pull request purpose

add eoan build

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -153,6 +154,69 @@ jobs:
...
+  #         1. docker create --name mobydick IMAGE CMD        <=> create a container (= instance of image) but container is NOT yet started
+  #         2. docker cp -a ${TRAVIS_BUILD_DIR} mobydick:/tmp <=> copy git repository (CI worker, checkout-ed on PR branch) into the container
+  #                                                               note: docker-cp works only if copy from/to containers (not images)
+  #         3. docker start -a mobydick                       <=> start to run the container (initialized with docker-cp)
+  - stage: eoan
+    dist: xenial
+    script: |
+      sudo docker pull ubuntu:eoan                                                                                    \
+      &&                                                                                                                \
+      sudo docker create --name mobydick ubuntu:eoan /bin/bash -c                                                     \
```
Comment (sylvestre) [inline .travis.yml]: Lot of duplication here, could you please try to make this better? thanks

**Comment by fghoussen** [comment]
merge ?

**Comment by sylvestre** [comment]
Please create a script in scripts/travis_ubuntu.sh to get all the info in a script
and run it with bash -v it wil get the same results without code duplication

File: .travis.yml (hunk 1 above)
Comment (sylvestre) [reply .travis.yml]: Could you please replicate what @dschwoerer did for fedora ? thanks

**Pull request closed without merging**